.cirrus.yaml

```diff
@@ -35,7 +42,7 @@ clone_script_template: &CLONE_SCRIPT_TEMPLATE
     fi
 
 chart_testing_task:
-  only_if: $CIRRUS_PRERELEASE != "true" && $CIRRUS_BRANCH != "master"
+  <<: *ONLY_ON_NONE_RELEASE_DRAFT_TEMPLATE
```

Comment:
small typo here, should it be `non-release` instead of `nonE-release` ?

Reply:
Interestingly I was trying to understand what NONE_RELEASE meant, since I just reused what was already available in the file. I will update that!